indico/web/client/styles/design_system/variables.scss

```diff
@@ -38,6 +39,22 @@ body {
   --text-brown-color: #60331f;
   --text-orange-color: #b73701;
 
+  --surface-green-color: #12f36c;
+  --surface-green-secondary-color: #2af87d;
+  --text-on-green-color: #000;
+  --surface-red-color: #e60f0f;
+  --surface-red-secondary-color: #b60404;
+  --text-on-red-color: #fff;
+  --surface-yellow-color: #ffba0c;
+  --surface-yellow-secondary-color: #ffbc37;
+  --text-on-yellow-color: #000;
+  --surface-organge-color: #f80;
+  --surface-organge-secondary-color: #ae6512;
```

Comment:
```suggestion
  --surface-orange-color: #f80;
  --surface-orange-secondary-color: #ae6512;
```

Reply:
Ah, those are copied over the from session bar fix PR by mistake. I'll remove them. Tho if you're going to be merging the session bar fix soon, I can just close this one.